res/sample/configuration.yaml

```diff
@@ -0,0 +1,182 @@
+          ReturnInputData: "false"  # For chained multi destination exports if true, passing the input data to next export function
+          HeaderName: "" # Name of the header key to add to the HTTP header
+          SecretValueKey: "" # Name of the secret for the header value in the SecretStore
+          SecretName: "" # Path to the secret for the header value in the SecretStore
+      MQTTExport:
+        Parameters:
+          BrokerAddress: "tcps://localhost:8883"
+          # Topic can have place holders in the form "{key-name}" that are replaced with matching Context values. Error occurs if no value match place holder(s).
+          Topic: "mytopic"
+          SecretName: "mqtt"
+          ClientId: "myclientid"
+          QOS: "0"
+          AutoReconnect: "false"
+          KeepAlive: "" # Empty value means use default setting
+          ConnectionTimeout: "" # Empty value means use default setting
+          Retain: "false"
+          SkipVerify: "false"
+          AuthMode: "none"
+          PersistOnError: "false"
+
+# InsecureSecrets are required for Store and Forward DB access and for authenticated HTTP exports and MQTT exports with
+# new MQTTSecretSend function when not using security services, i.e. Vault
+  InsecureSecrets:
+    http:
+      SecretName: "http"
+      SecretData:
+        headervalue: "<put your value here>"
+    AES:
+      SecretName: "aes"
+      SecretData:
+        key: "217A24432646294A404E635266556A586E3272357538782F413F442A472D4B6150645367566B59703373367639792442264529482B4D6251655468576D5A7134"
+
+Service:
+  Host: "localhost"
+  Port: 59700
+  StartupMsg: "app-sample has Started"
+
+# Note Core Metadata is the only required client as it is now used for the version check
+# The other clients are only needed if the client interface is being used by one of the configured functions.
+Clients:
+  # Also used for Device Service, DeviceProfile and Device clients, which are not currently used by any configurable functions
+
+  # Used for Event client which is used by PushToCoreData function
+  core-data:
+    Protocol: "http"
+    Host: "localhost"
+    Port: 59880
+
+  # Used for Command client, which is not currently used by any configurable functions
+  core-command:
+  #   Protocol: "http"
+  #   Host: "localhost"
+  #   Port: 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
+    UseMessageBus: true
+
+  # Used for Notification and Subscription client, which are not currently used by any configurable functions
+  support-notifications:
+    Protocol: "http"
+    Host: "localhost"
+    Port: 59860
+# uncomment when running from command-line in hybrid mode with -cp -o flags
+#   core-metadata: # Used for version check on start-up comes from Common Config
```

Comment:
I realized this is confusing to have since now of the configurable functions use these clients, so needs to be removed
```suggestion
# uncomment when running from command-line in hybrid mode with -cp -o flags
#Clients:
#   core-metadata: # Used for version check on start-up comes from Common Config
```